feature-libs/product-configurator/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.ts

```diff
@@ -116,9 +122,18 @@ export class ConfiguratorAttributeProductCardComponent
 
   onHandleDeselect(): void {
     this.loading$.next(true);
-    this.handleDeselect.emit(
-      this.productCardOptions.productBoundValue.valueCode
-    );
+    {
+      if (
+        this.productCardOptions?.productBoundValue?.selected &&
```

Comment:
```suggestion
        this.productCardOptions.productBoundValue.selected &&
```
ProductCardOptions is mandatory